tests/testsuite/global_cache_tracker.rs

```diff
@@ -2004,7 +2004,16 @@ fn compatible_with_older_cargo() {
     assert_eq!(get_registry_names("src"), ["middle-1.0.0", "new-1.0.0"]);
     assert_eq!(
         get_registry_names("cache"),
-        ["middle-1.0.0.crate", "new-1.0.0.crate", "old-1.0.0.crate"]
+        // Duplicate crates from two different cache location
```

Comment:
I am not sure if garbage collection should support cases like this.
Ideal it should?

Reply:
cc @ehuss

Reply:
Actually it is doing its job correctly, so no need to worry about changing these tests.